packages/remix-server-runtime/server.ts

```diff
@@ -406,60 +410,79 @@ async function singleFetchAction(
       signal: request.signal,
       ...(request.body ? { duplex: "half" } : undefined),
     });
-    let result = await staticHandler.queryRoute(handlerRequest, {
```

Comment:
`action`'s use `query` now as well and pass `skipLoaders` to just call the action.  I think this _should_ let us combine the `singleFetchAction`/`singleFetchLoaders` functions eventually too - but didn't look into that yet as part of this work